archlinux/PKGBUILD-pipewire-qubes.install

```diff
@@ -0,0 +1,15 @@
+## arg 1:  the new package version
+post_install() {
+  systemctl --global --no-reload preset pipewire.service pipewire.socket wireplumber.service || :
+  qvm-features-request supported-service.pipewire=1
```

Comment:
This is not the way to do it (nor in other package types). Create a file in `/etc/qubes/post-install.d` for it. See https://dev.qubes-os.org/projects/core-admin/en/latest/qubes-features.html#services or existing scripts there for details.

Reply:
In this repo, there is already `appvm-scripts/etc/qubes/post-install.d` for it.